templates/ts-apollo-fullstack/server/src/db.ts

```diff
@@ -17,3 +17,11 @@ export const createDB = async () => {
     const db = knex(generateConfig.dbmigrations)
     return db
 }
+
+export const createMongoDB = async () => {
+    const MongoClient = require('mongodb').MongoClient;
+    const url = "mongodb url";
```

Comment:
This should not be merged in the template. Maybe we should have a new template for MongoDB

Reply:
Resolver generator always generates knex stuff so I need to do a little bit refactor to enable swapping datasources easily without creating separate templates for them. Also yes. This needs to be removed - I'm trying to figure out the best way to generate createRuntimeContext method with different DataSources and Services